extensions/prost/private/prost.bzl

```diff
@@ -453,6 +457,10 @@ rust_prost_toolchain = rule(
             doc = "Whether to include transitive dependencies. If set to True, all transitive dependencies will directly accessible by the dependent crate.",
             default = False,
         ),
+        "is_nostd": attr.bool(
```

Comment:
nit, can this be renamed to `is_no_std` to match the `no_std` feature?

Reply:
Done :smile: